app/models/record/poi_record.rb

```diff
@@ -39,17 +39,14 @@ def convert_xml_to_hash
   end
 
   def parse_single_poi_from_xml(poi)
-    {
+    poi_data = {
       id: poi.attributes["id"].try(:value),
       name: poi.attributes["name"].try(:value),
       description: poi.xpath("description/text").try(:text),
       mobile_description: poi.xpath("descriptionMobileSingle/text").try(:text),
-      categories: parse_categories(poi),
```

Comment:
categories fliegt hier raus?

Reply:
> cleanup fields without graphql mapping

@endorfin was heißt hier "kein graphql mapping"?

Reply:
Felder die nicht vom GraphQL unterstützt werden, fliegen raus

Reply:
es gab bisher keine "categories" in GraphQL.

Reply:
danke euch